app/src/main/java/com/perflyst/twire/chat/ChatEmoteManager.java

```diff
@@ -33,10 +34,12 @@
 
     private final String channelName;
     private final int channelId;
+    private final ChatEmoteSettings emote_settings;
 
-    ChatEmoteManager(String channelName, int channelId) {
+    ChatEmoteManager(String channelName, int channelId, ChatEmoteSettings emotesettings) {
```

Comment:
Sorry, after reviewing this again I realized that there is an even easier solution. Just pass the settings along from the ChatManager.
```suggestion
    ChatEmoteManager(String channelName, int channelId, Settings settings) {
```

Reply:
Done in 9ac85cef6e545d9bc5ea01b033c01d6ca5fa9375